test/NuGet.Core.FuncTests/Msbuild.Integration.Test/MsbuildRestoreTaskTests.cs

```diff
@@ -1195,7 +1195,7 @@ public async Task MsbuildRestore_PackagesConfigDependency_WithHttpSource_Warns(b
         [InlineData("", true)]
         [InlineData("true", false)]
         [InlineData("TRUE", false)]
-        public async Task MsbuildRestore_PackagesConfigDependency_WithHttpSourceAndAllowInsecureConnections_WarnsCorrectly(string allowInsecureConnections, bool hasHttpWarning)
+        public async Task MsbuildRestore_PackagesConfigDependency_WithHttpSourceAndAllowInsecureConnections_ErrorsCorrectly(string allowInsecureConnections, bool hasHttpWarning)
```

Comment:
I know you didn't name this, but I'm confused once again whether this is testing AllowInsecureConnections as True or not. Why would it ever "ErrorCorrectly" if `AllowInsecureConnections` is true? The answer seems to be that an empty string is passed in, which I presume defaults to false.

Consider changing either the tests overall to assert the error case and then the success case (my preference), or see if there's a better name for this test to be clearer.

Reply:
I have made it into two test cases to make the naming simpler and the test clearer.